paper-server/src/main/java/org/bukkit/craftbukkit/block/CraftVault.java

```diff
@@ -24,4 +39,96 @@ public CraftVault copy() {
     public CraftVault copy(Location location) {
         return new CraftVault(this, location);
     }
+
+    @Override
+    public double getActivationRange() {
+        return this.getSnapshot().getConfig().activationRange();
+    }
+
+    @Override
+    public void setActivationRange(final double activationRange) {
+        Preconditions.checkArgument(Double.isFinite(activationRange), "deactivation range must be a number");
```

Comment:
this message maybe can be more specific about the infinite rather than just "number"?